tannerb9/PRIX-client
# Read Me

# Description
Included instructions on ReadMe on how to set up and run the app.

## Type of change
- [x] ReadMe instructions

# Testing Instructions
Follow the ReadMe instructions to ensure all steps provided get the app to run successfully.

# Checklist:
